### PR TITLE
DOC: Add example to ``rng.beta(...)``

### PR DESCRIPTION
**Pull request by linus-md**
Relates to #21351. See also: #25349.

### CODE REVIEW

File: numpy/random/_generator.pyx [hunk 1]
```diff
@@ -396,6 +396,41 @@ cdef class Generator:
...
+        whether ``a`` or ``b`` is greater. The distribution is mirror
+        symmetric. See for example:
+        
+        >>> a, b, size = 2, 7, 10000
+        >>> sample_left = rng.beta(a=a, b=b, size=size)
+        >>> sample_right = rng.beta(a=b, b=a, size=size)
+        >>> m_left, m_right = np.mean(sample_left), np.mean(sample_right)
+        >>> m_left, m_right
+        (0.22334081861904062, 0.7749320166133927)  # may vary
+        >>> m_left + m_right
```
Comment (eendebakpt) [inline numpy/random/_generator.pyx:420]: Why show this value? I would either omit it, or add with a comment we expect this to be 1.0 (up to statistical fluctuations). Similar for the previous line (where we expect 2/9, 7/9)

Comment (linus-md) [reply numpy/random/_generator.pyx]: I made a few changes do you like it more the way it is now?

File: numpy/random/_generator.pyx [hunk 2]
```diff
@@ -396,6 +396,43 @@ cdef class Generator:
...
+        >>> m_left, m_right = np.mean(sample_left), np.mean(sample_right)
+        >>> print(m_left, m_right)
+        0.2238596793678923 0.7774613834041182  # may vary
+        >>> print(m_left - a/(a+b))
+        0.001637457145670096  # may vary
+        >>> print(m_right - b/(a+b))
+        -0.0003163943736596009  # may vary
+
+        Display the histogram of the two samples:
+        >>> import matplotlib.pyplot as plt
```
Comment (mattip) [inline numpy/random/_generator.pyx:426]: I think you need a blank line , see the [resulting documentation](https://output.circle-artifacts.com/output/job/eed957a2-ec55-4894-bf37-20eafda82247/artifacts/0/doc/build/html/reference/random/generated/numpy.random.Generator.beta.html#numpy.random.Generator.beta)

Comment (linus-md) [reply numpy/random/_generator.pyx:427]: Thanks.I should have spotted that. I added an empty line

**Comment by mattip** [comment]
Thanks @linus-md

**Pull request merged**